editor/src/messages/tool/tool_messages/spline_tool.rs

```diff
@@ -360,8 +360,6 @@ impl Fsm for SplineToolFsmState {
 				tool_options.stroke.apply_stroke(tool_data.weight, layer, responses);
 				tool_data.current_layer = Some(layer);
 
-				responses.add(Message::StartBuffer);
```

Comment:
This removal is different from all the other tool code changes, which besides this one all add a DeferMessage::AfterGraphRun call. Just bringing attention to this in case it's a bug. Otherwise this PR LGTM.